src/controllers/__tests__/sessionController.test.ts

```diff
@@ -142,21 +144,32 @@ describe("SessionController#listSessions", () => {
   beforeEach(() => {
     jest.clearAllMocks();
     mockSmembers.mockImplementationOnce((_, callback) => {
-      callback(null, [
-        `USER-${mockedUser.fiscal_code}-SESSION-${mockedUser.session_token}`
-      ]);
+      callback(null, [`SESSIONINFO-${mockedUser.session_token}`]);
     });
     mockMget.mockImplementationOnce((_, callback) => {
       callback(null, [JSON.stringify(expectedSessionInfo)]);
     });
   });
   it("returns list of sessions for an authenticated user", async () => {
     req.user = mockedUser;
+    mockSmembers.mockImplementationOnce((_, callback) => {
+      callback(null, [`SESSIONINFO-${mockedUser.session_token}`]);
+    });
```

Comment:
As for above comment, this can be deleted

Reply:
fixed in f13d975